src/main/java/org/cryptomator/cryptofs/health/dirid/OrphanDir.java

```diff
@@ -114,32 +102,40 @@ public void fix(Path pathToVault, VaultConfig config, Masterkey masterkey, Crypt
 		Files.delete(orphanedDir);
 	}
 
+	CryptoPathMapper.CiphertextDirectory prepareCryptoFilesystem(Path pathToVault, FileNameCryptor cryptor, String clearStepParentDirName) throws IOException {
+		Path dataDir = pathToVault.resolve(Constants.DATA_DIR_NAME);
+		String rootDirHash = cryptor.hashDirectoryId(Constants.ROOT_DIR_ID);
+		Path vaultCipherRootPath = dataDir.resolve(rootDirHash.substring(0, 2)).resolve(rootDirHash.substring(2)).toAbsolutePath();
+
+		//create  if not existent with constant id
+		String cipherRecoveryDirName = convertClearToCiphertext(cryptor, Constants.RECOVERY_DIR_NAME, Constants.ROOT_DIR_ID);
+		Path cipherRecoveryDirFile = vaultCipherRootPath.resolve(cipherRecoveryDirName + "/" + Constants.DIR_FILE_NAME);
+		Files.createDirectory(cipherRecoveryDirFile.getParent());
```

Comment:
This will probably fail if it already exists.

Reply:
This case should also be covered by a unit test

Reply:
Fixed in 1d0be7d